prime-router/src/testIntegration/resources/datatests/translation-test-config.csv

```diff
@@ -20,16 +20,8 @@ HL7_to_FHIR_to_HL7/elims_2_52_5068110_35230_NoPII.hl7,,HL7_to_FHIR_to_HL7/elims_
 HL7_to_FHIR_to_HL7/elims_2_72_3029198209_5121_NoPII.hl7,,HL7_to_FHIR_to_HL7/elims_2_72_3029198209_5121_NoPII-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","test.dev-elims","","",""
 HL7_to_FHIR_to_HL7/elims_29_5065302_35227_NoPII_CANCELED.hl7,,HL7_to_FHIR_to_HL7/elims_29_5065302_35227_NoPII_CANCELED-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","test.dev-elims","","",""
 HL7_to_FHIR_to_HL7/hci.hl7,,HL7_to_FHIR_to_HL7/hci-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","test.dev-hci","","",""
-FHIR_to_HL7/sample_AOE_1_20230220-0001.fhir,,FHIR_to_HL7/sample_AOE_1_20230220-0001-custom-datetime.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/original-pipeline-transforms",test.DEV_FULL_ELR,""
-FHIR_to_HL7/sample_AOE_1_20230220-0001.fhir,,FHIR_to_HL7/sample_AOE_1_20230220-0001.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/original-pipeline-transforms","",""
-FHIR_to_HL7/sample_AOE_2_20230220-0001.fhir,,FHIR_to_HL7/sample_AOE_2_20230220-0001.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/original-pipeline-transforms","",""
 FHIR_to_HL7/sample_SR_1_20230302-0001.fhir,,FHIR_to_HL7/sample_SR_1_20230302-0001.hl7,,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform",ca-test.default-receiver,""
 FHIR_to_HL7/sample_SR_1_20230302-0001-missing-data.fhir,,FHIR_to_HL7/sample_SR_1_20230302-0001-missing-data.hl7,,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform",ca-test.default-receiver,""
 FHIR_to_HL7/sample_SR_2_20231024-0002.fhir,,FHIR_to_HL7/sample_SR_2_20231024-0002.hl7,metadata/hl7_mapping/receivers/STLTs/NY/NY-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","",""
 FHIR_to_HL7/sample_SR_2_20231106-0003.fhir,,FHIR_to_HL7/sample_SR_2_20231106-0003.hl7,metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","",""
-FHIR_to_HL7/sample_RADx_MARS_20230406-0001.fhir,,FHIR_to_HL7/sample_RADx_MARS_20230406-0001.hl7,metadata/hl7_mapping/ORU_R01/RADx_MARS/RADx_MARS-base,HL7,PASS,"","","","",""
```

Comment:
This translation in not in use and the expectation is that this is not how we will need to handle the RADxMARS data